### PR TITLE
Feat : add toast

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

File: Components/toast.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    display: flex;
+    justify-content: space-between;
+    font-size: 14px;
+    max-width: 400px;
+    font-weight: 500;
+    align-items: center;
+    border-radius: 5px;
+    box-shadow: 0 3px 5px -1px rgb(0 0 0 / 20%), 0 6px 10px 0 rgb(0 0 0 / 14%), 0 1px 18px 0 rgb(0 0 0 / 12%);
+}
+
```
Comment (ZuberDunge) [inline Components/toast.css:16]: this is for showcase version

File: Components/toast.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    gap: 0.5rem;
+    align-items: center;
+}
+
+.bat-retry span {
+    color: var(--white-only);
+}
+
+
+#bat-toast {
```
Comment (ZuberDunge) [inline Components/toast.css:30]: this is for live demo version

**Review by prachi-sahani — COMMENTED** [review]
Code looks good to me 👍🏻

**Review by siddhigate — APPROVED** [review]
Gtg

**Pull request merged**